src/components/Frame/Frame.tsx

```diff
@@ -177,11 +177,16 @@ class FrameInner extends React.PureComponent<CombinedProps, State> {
         </div>
       ) : null;
 
+    const contextualSaveBarClassName = classNames(
+      styles.ContextualSaveBar,
+      unstableGlobalTheming && styles['ContextualSaveBar-globalTheming'],
```

Comment:
Any reason why you're not simply using `.globalTheming`?

Reply:
Something we're going to be removing at some point so I didn't put much thought behind it 🤷‍♂